### PR TITLE
chore: remove tracking from sample apps

### PR DESCRIPTION
**Pull request by Shahroz16**
closes: [MBL-338 : Track In-App Metrics](https://linear.app/customerio/issue/MBL-338/track-in-app-metrics)

### CODE REVIEW

File: messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistSdk.kt [hunk 1]
```diff
@@ -60,7 +60,7 @@ object GistSdk {
     @JvmStatic
     fun getInstance() = this
 
-    private fun onActivityResumed(activity: Activity) {
+    internal fun onActivityResumed(activity: Activity) {
```
Comment (Shahroz16) [inline messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistSdk.kt:63]: need to make sure app is compiling

File: samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/ui/login/LoginViewModel.kt [hunk 2]
```diff
@@ -41,6 +43,7 @@ class LoginViewModel @Inject constructor(
         loadData()
     }
 
+    @RequiresApi(Build.VERSION_CODES.FROYO)
```
Comment (mrehan27) [inline samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/ui/login/LoginViewModel.kt:46]: Why is this needed? 🤔

**Review by mrehan27 — APPROVED** [review]
